Tests/SonarQube.TeamBuild.PreProcessor.Tests/RoslynExportProfileTests.cs

```diff
@@ -85,20 +94,31 @@ private static RoslynExportProfile LoadAndCheckXml(string xml)
         private static void AssertExpectedAdditionalFileExists(string fileName, RoslynExportProfile profile)
         {
             Assert.IsNotNull(profile.Configuration.AdditionalFiles);
-            IEnumerable<AdditionalFile> matches = profile.Configuration.AdditionalFiles.Where(f => string.Equals(fileName, f.FileName, System.StringComparison.OrdinalIgnoreCase));
-            Assert.AreNotEqual(0, matches.Count(), "Expected additional file was not found. File name: {0}", fileName);
-            Assert.AreEqual(1, matches.Count(), "Expecting only one matching file. File name: {0}", fileName);
-            Assert.IsNotNull(matches.First().Content, "File content should not be null. File name: {0}", fileName);
+            AdditionalFile[] matches = profile.Configuration.AdditionalFiles.Where(f => string.Equals(fileName, f.FileName, System.StringComparison.OrdinalIgnoreCase)).ToArray();
+            Assert.AreNotEqual(0, matches.Length, "Expected additional file was not found. File name: {0}", fileName);
+            Assert.AreEqual(1, matches.Length, "Expecting only one matching file. File name: {0}", fileName);
+            Assert.IsNotNull(matches[0].Content, "File content should not be null. File name: {0}", fileName);
```

Comment:
Unrelated changes. Using arrays makes inspecting values when debugging easier (they're indexable and the content is rendered better in the debug visualiser).